src/Sulu/Bundle/PreviewBundle/Preview/Exception/TwigException.php

```diff
@@ -17,29 +17,29 @@
 class TwigException extends PreviewRendererException
 {
     /**
-     * @var \Twig_Error
+     * @var \Exception
```

Comment:
Would you really change the scope of this exception? Not finding a valid template is not really `TwigException`, since twig hasn't even been loading yet, don't you think?